packages/graphql/src/configs/localhost.js

```diff
@@ -14,6 +14,7 @@ const config = {
   ipfsRPC: `http://${HOST}:5002`,
   bridge: 'https://bridge.dev.originprotocol.com',
   notifications: `http://${HOST}:3456`,
+  //growth: 'http://localhost:4001',
```

Comment:
commented line here

Reply:
Yep. If it's ok I'll leave it in there. It's handy for people who run the growth server locally.